main.tf

```diff
@@ -10,6 +10,8 @@ resource "aws_s3_bucket" "default" {
   #checkov:skip=CKV_AWS_18: "Logging handled in logging configuration resource"
   #checkov:skip=CKV_AWS_21: "Versioning handled in Versioning configuration resource"
   #checkov:skip=CKV_AWS_145: "Encryption handled in encryption configuration resource"
+  # Up to customer whether they have notifications enabled.
```

Comment:
This option is currently not supported in this module. The fix should be to allow to opt in for it by passing the sns topic arn.